docs/mcf_format.md

```diff
@@ -288,10 +288,12 @@ other nodes can be `typeOf`.
 
 ### Template MCF
 
-Template MCF is an intermediary MCF that we use for mapping datasets to instance
-MCF. Template MCF does not get ingested into the Data Commons graph.
+Template MCF, or tMCF, is an intermediary MCF that we use for mapping datasets to instance
+MCF. Template MCF does not get ingested into the Data Commons graph. They are
+referencing the data in a paired .csv and together will be used to generate a
```

Comment:
nit: "They are referencing the data in a" -> "Template MCF references data in a"